contributors/devel/sig-architecture/conformance-tests.md

```diff
@@ -154,6 +154,43 @@ To promote a test to the conformance test suite, open a PR as follows:
     all of the required [conformance test comment metadata]
 - add the PR to SIG Architecture's [Conformance Test Review board]
 
+#### Windows nodes considerations
+
+Not all tests are able to pass on Windows, some of which have been labeled as
+``[LinuxOnly]``. The main reasons are defined in the Windows Node Support KEP
+(https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/20190103-windows-node-support.md#what-will-never-work) . The most common reasons found why tests cannot pass on Windows are:
+
+- Pod ``SecurityContext`` set. Most of the fields are Linux specific, and with
+  and any field set in the Pod's ``SecurityContext`` will result in the Pod not
+  being able to spawn, or not work as intended.
+- File permissions cannot be set on volumes. Tests which are setting Volumes
+  ``DefaultMode`` or ``Mode`` and expecting those file permissions to be set will fail.
+- Individual files cannot be mounted on Windows Containers. Tests which are mounting
+  or expecting such files to be mounted (including ``/etc/hosts``, ``/etc/resolv.conf``,
+  ``/dev/termination-log``).
+- Tests which create EmptyDir volumes with ``Medium=memory``. The volumes are created
+  as regular EmprtyDir volumes, and if the test checks if the fstype is ``tmps``, it will
+  fail.
+- Pod ``HostNetwork=true`` set. Is not supported on Windows, and if set, the Pod will
+  fail to spawn.
+- Cannot resolve PQDNs (Partially Qualified Domain Names). At the moment, Windows Containers are
+  configured with a single DNS suffix (e.g.: ``lish.svc.cluster.local``). The containers are
+  only able to resolve FQDNs, or names within namespace (e.g.: ``foo.lish.cluster.local``). Anything
+  in between it cannot (e.g.: ``foo.lish``), and tests relying on PQDNs will fail.
+- Running Linux specific commands. Most Windows images used in testing are based on busybox, and thus,
+  quite a few Linux commands are acceptable. But, due to the differences between Linux and Windows, not
+  all commands and command options are included in Windows busybox binary. The most concerning cases are:
```

Comment:
The commands found in the Windows busybox binary are:

    Currently defined functions:
        [, [[, ar, ash, awk, base64, basename, bash, bunzip2, bzcat, bzip2, cal, cat, catv, chmod, cksum, clear, cmp, comm, cp, cpio,
        cut, date, dc, dd, df, diff, dirname, dos2unix, dpkg-deb, du, echo, ed, egrep, env, expand, expr, false, fgrep, find, fold,
        ftpget, ftpput, getopt, grep, groups, gunzip, gzip, hd, head, hexdump, id, ipcalc, kill, killall, less, ln, logname, ls,
        lzcat, lzma, lzop, lzopcat, man, md5sum, mkdir, mktemp, mv, nc, od, patch, pgrep, pidof, printenv, printf, ps, pwd, rev, rm,
        rmdir, rpm2cpio, sed, seq, sh, sha1sum, sha256sum, sha3sum, sha512sum, shuf, sleep, sort, split, stat, strings, sum, tac,
        tail, tar, tee, test, touch, tr, true, truncate, uname, uncompress, unexpand, uniq, unix2dos, unlink, unlzma, unlzop, unxz,
        unzip, usleep, uudecode, uuencode, vi, wc, wget, which, whoami, whois, xargs, xz, xzcat, yes, zcat

I wonder if it's worth adding them here.